src/lib/coda_networking/coda_networking.ml

```diff
@@ -345,6 +393,10 @@ module Make (Inputs : Inputs_intf) = struct
     Gossip_net.query_peer t.gossip_net peer
       Rpcs.Transition_catchup.dispatch_multi state_hash
 
+  let prove_ancestry t peer input =
```

Comment:
but this should be verify_ancestry if I'm getting my sides of the protocol right